gameplay-editor/src/ProjectWizard.cpp

```diff
@@ -134,6 +128,7 @@ void ProjectWizard::onNewProjectPressed()
 
 void ProjectWizard::onOpenProjectPressed()
 {
+    resetEditor();
```

Comment:
Do you want to have this here? What if open project was invoked, nothing is selected (or is selected but is null or doesn't exist), and it returns without invoking `projectOpened`? The editor has been reset. All prior work is gone.

At least that's what I see just from reading the code.